### DISCUSSION TITLE
OWASP Top 10 Categories

### DISCUSSION THREAD

**Discussion by pfg-jmbatt** [github/codeql#14728] [Ideas]
Are there any plans to add the OWASP Top 10 categories to the CodeQL findings so vulnerabilities can be grouped by the respective OWASP Top 10 Category when applicable?

**Comment by aeisenberg** [comment]
Thanks for your question. We have no plans to integrate this directly into CodeQL, but we are considering updating our docs to make the relationship between queries and OWASP categories clearer. Note that this is a consideration, not a promise. 😄
